### PR TITLE
Create vespa.json

### PR DESCRIPTION
**Pull request by HarshMN2345**
Implemented Vespa Search
Associated issue #692

To utilize this functionality, follow these steps:

Start Vespa Instance:
Ensure you have a running Vespa instance. If not, refer to Vespa's official documentation for installation instructions.

Add JSON File to Swirl Configuration:
Insert the provided JSON file into your Swirl configuration file.

Restart Swirl:
Restart Swirl to apply the changes.

Now, you can start searching for Vespa content. Use the following command:

swirl search vespa "my query"

### CODE REVIEW

**Comment by HarshMN2345** [comment]
@sidprobstein please review it

**Comment by HarshMN2345** [comment]
@dnicodemus pls review it

**Comment by sid-swirl** [comment]
This looks awesome @HarshMN2345 ... I believe you should remove the space from the result_mappings. Can you add author, date_published and url also?

Also, does Vespa return the  # found, # retrieved? If possible, add those to the response_mappings as noted here: https://docs.swirl.today/2.-User-Guide.html#response-mappings

If you can post a vespa JSON response, we can help with these! Thank you again!

**Comment by HarshMN2345** [comment]
@sid-swirl  i have done the changes as asked

File: SearchProviders/vespa.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  "url": "https://localhost:8080/search/",
+  "query_template": "{url}?query={query_string}&fields={fields}&hits={hits}",
+  "query_processors": ["AdaptiveQueryProcessor"],
+  "query_mappings": "",
+  "result_processors": [
+    "MappingResultProcessor",
+    "LenLimitingResultProcessor",
+    "CosineRelevancyResultProcessor"
+  ],
+  "response_mappings": "RESULTS=root_object.hits.hits, #found=root_object.hits.found, #retrieved=root_object.hits.retrieved",
```
Comment (sidprobstein) [inline SearchProviders/vespa.json:15]: Hi @HarshMN2345 ... this is good, however, #found and #retrieved are not legal. They should be FOUND and RETRIEVED

**Review by sidprobstein — CHANGES_REQUESTED** [review]
Please address commented change thank you

**Review by sidprobstein — CHANGES_REQUESTED** [review]
Also, please re-target this to swirlai:develop branch as per the guidelines. Thank you.

https://swirl.today/contribute-to-swirl-this-hacktoberfest-2023/

**Comment by HarshMN2345** [comment]
@sid-swirl okay i am making a new pr for the changes and closing this one

**Pull request closed without merging**